### PR TITLE
Add faq 1 & 2

### PR DESCRIPTION
**Pull request by s950449**
Add faqs 1&2
Faq 1 is about Python version
Faq 2 is about task 8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,8 +227,14 @@ Some examples:
 * Answer an issue that could not be solved easily (10%)
 
 * Send a PR and solve a bug in this repository (30%)
-
-
+###FAQ
```
Comment (amjltc295) [inline README.md]: This should be `##`

File: README.md [hunk 2]
```diff
@@ -227,8 +227,14 @@ Some examples:
...
+1.Please make sure that Python3 version is 3.5 or later.
+You could use `python3 -V` to check your Python version.
```
Comment (amjltc295) [inline README.md]: This is a bit misleading. Using Conda is suggested; you don't have to specify 3 here if you are in the Conda environment. Could you add a Conda solution as a suggested approach?

File: README.md [hunk 3]
```diff
@@ -227,8 +227,14 @@ Some examples:
...
+2.In order to pass Task 8, please make sure that you have already install the font 'arial' or arial.ttf has already in your system's font folder.
+If you are using Debian/Ubuntu, you can install the font 'arial' by using apt:
+`sudo apt install ttf-mscorefonts-installer`
+`sudo fc-cache`
+To check whether the font has successfully installed, you can use `fc-match Arial`
```
Comment (amjltc295) [inline README.md]: This should work after installing Conda the environment, did it fail to work? If it fails, we should modify the default font in `utils.py`.

File: README.md (hunk 1 above)
Comment (s950449) [reply README.md]: OK

**Pull request closed without merging**